docs/Deployment.md

```diff
@@ -91,7 +85,6 @@ RedisRaft is indirectly affected by Redis's own configuration, and there are sev
 | save                   | ""             | RedisRaft uses the RDB file as its own snapshot and manages the BGSAVE operation, so Redis needs to be configured in a way that does not interfere. |
 | dbfilename             | user defined   | You are free to configure the filename as desired. |
 | replicaof              | <none>         | RedisRaft implements its own replication mechanism; traditional Redis replication may not be enabled. |
-| requirepass            | <none>         | RedisRaft cluster nodes exchange messages on the same Redis port that the client uses; however, RedisRaft does not implement authentication. |
```

Comment:
We have `cluster-user` and `cluster-password` config now. RR should work with `requirepass`.